src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
+    /**
+     * main method to run the program
+     *
+     * @param args command line arguments taken in
+     */
```

Comment:
Minor nitpick but perhaps consider starting Javadoc sentences with an uppercase letter and ending with a full stop to keep within the coding standard.